site/content/blog/2021-08-01-whats-new-in-svelte-august-2021.md

```diff
@@ -14,7 +14,6 @@ From The Changelog ([JS Party Ep. 182](https://changelog.com/jsparty/182)) to Sv
 For a full list of features and bug fixes, check out the [Svelte changelog](https://github.com/sveltejs/svelte/blob/master/CHANGELOG.md).
 
 ## SvelteKit Updates
```

Comment:
This is a good list! Some other PRs of note might be:
- https://github.com/sveltejs/kit/pull/2007
- https://github.com/sveltejs/kit/pull/1827
- https://github.com/sveltejs/kit/pull/1890
- https://github.com/sveltejs/kit/pull/1984

Reply:
I'm leaving fixes out for now, unless they introduce new features. Added the top 2 though :)